js/main.js

```diff
@@ -100,9 +103,44 @@ var buildFragment = function (array) {
 };
 
 // shows user posts -----------------------------------------------------------
-var showPosts = function () {
+var showPosts = function (array) {
   var postedPics = document.querySelector('.pictures');
-  postedPics.appendChild(buildFragment(shuffleArray(generateData(POSTS_COUNT))));
+  postedPics.appendChild(buildFragment(array));
+};
+
+showPosts(userPosts);
+
+// adds cooments to popup with big photo ----------------------------------------
+var renderComments = function (array) {
+  var fragment = document.createDocumentFragment();
+  var commentsBlock = document.querySelector('.social__comments');
+  var commentTemplate = commentsBlock.querySelector('.social__comment');
+  commentsBlock.textContent = '';
+  array.forEach(function (item) {
+    var commentElement = commentTemplate.cloneNode(true);
+    commentElement.querySelector('.social__picture').src = item.avatar;
+    commentElement.querySelector('.social__picture').alt = item.name;
+    commentElement.querySelector('.social__picture').width = '35';
+    commentElement.querySelector('.social__picture').height = '35';
+    commentElement.querySelector('.social__text').textContent = item.message;
+    fragment.appendChild(commentElement);
+  });
+  return fragment;
+};
+
+// shows popup with big photo  --------------------------------------------------
+var showPopupPic = function (array) {
+  var bodyModalOpen = document.querySelector('BODY');
```

Comment:
лучше писать в одном стиле
```suggestion
  var bodyModalOpen = document.querySelector('body');
```